core/safe_refcount.h

```diff
@@ -100,40 +100,77 @@ static _ALWAYS_INLINE_ T atomic_exchange_if_greater(volatile T *pw, volatile V v
 // GCC guarantees atomic intrinsics for sizes of 1, 2, 4 and 8 bytes.
 // Clang states it supports GCC atomic builtins.
 
+/* Clang and GCC>4.7.4 can use __atomic */
+#define USE__ATOMIC
+
+#if !defined(__clang__)
+#define GCC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
+#if GCC_VERSION <= 40704
```

Comment:
We don't support GCC versions below 4.7.4 (we're about to migrate to C++11, which was not fully supported back then), so no need to keep the legacy code.

Reply:
I'm happy to hear that, it makes things simpler :) I (supposedly) also made travis-ci happier while there.